packages/calcite-components/src/assets/styles/includes.scss

```diff
@@ -210,3 +210,12 @@ $common-animatable-props: "background-color, block-size, border-color, box-shado
   transition-duration: var(--calcite-animation-timing);
   transition-timing-function: ease-in-out;
 }
+
+// Mixin for text highlighting styles.
+// - this should be used in conjunction with the `text.tsx` util.
+@mixin text-highlight-item() {
+  .text-match {
+    font-weight: var(--calcite-font-weight-bold);
```

Comment:
I think we need to set color as well since the default `mark` element defines a color. I had it set to inherit to take the color of the parent element.

Reply:
Excellent catch. Will fix!